subprojects/internal-testing/src/main/groovy/org/gradle/test/fixtures/file/TestFileHelper.groovy

```diff
@@ -291,8 +291,8 @@ class TestFileHelper {
         }
     }
 
-    private static boolean isUnix() {
-        return !System.getProperty('os.name').toLowerCase().contains('windows')
+    private static boolean isWindows() {
```

Comment:
This method's name confused me a little on MacOS.  It's checking for Windows, so let's name it that and invert the tests.

Reply:
🎉